### ISSUE TITLE
[import/named] Doesn't recognize named exports when requiring folders with index.js

### ISSUE THREAD

**Issue by JohnyDays** [benmosher/eslint-plugin-import#32]
* app |
  *  client.js
  *  middlewares |
       * index.js

````javascript
//index.js
export function test {}
````
````javascript
//client.js
import {test} from './middlewares'
````

Error: test not found in './middlewares' [import/named]

Not my actual real project, but this was the most minimal example i could find :+1:

**Comment by benmosher** [comment]
I added a test for this and could not reproduce, except... are the parentheses intentionally missing from your `test` function?

My test file is essentially
```js
// ./module/index.js
export function test() {}
```

**Comment by benmosher** [comment]
Are you using Sublime + SublimeLinter-eslint? There is a known issue with paths that requires a `.sublimelinterrc` to straighten out. I've just added info to the README in case this is relevant.

**Comment by JohnyDays** [comment]
Maybe it's something special about my setup, I'll try to setup a failing example on github when I get the time
